Core/AttendantUserServices/AttendantOnboardingService.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Threading.Tasks;
+using Data.AppDbContext;
+using Data.Dtos;
+using Data.Models;
+using Microsoft.Extensions.Logging;
+using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+
+namespace Core.AttendantUserServices
+{
+    public class AttendantOnboardingService : IAttendantOnboardingService
+    {
+        private readonly TeejayDbContext _context;
```

Comment:
Injecting DBContext into the controller is strongly prohibited. Please use it within services only. This is to enhance portability, reusability, sand scalability of services. Please urgently remove it.